tests/sentry/snuba/metrics/test_query_builder.py

```diff
@@ -1493,8 +1502,24 @@ def test_resolve_tags_with_binary_tuple(self):
         tags = [("/foo", "ios"), ("/bar", "android")]
 
         for transaction, platform in [("transaction", "platform")] + tags:
-            indexer.record(use_case_id=self.use_case_id, org_id=self.org_id, string=transaction)
-            indexer.record(use_case_id=self.use_case_id, org_id=self.org_id, string=platform)
+            indexer.record(
+                use_case_id=(
+                    UseCaseID.SESSIONS
+                    if self.use_case_id is UseCaseKey.RELEASE_HEALTH
+                    else UseCaseID.TRANSACTIONS
+                ),
```

Comment:
need more time to read over the whole approach, but the fact that you keep repeating this condition suggests to me that it should be in a method

Reply:
Fixed, good point

Reply:
Fixed now